docs/source/peer-chaincode-devmode.rst

```diff
@@ -0,0 +1,114 @@
+Using dev mode
+==============
+
+Normally chaincodes are started and maintained by the peer. However in “dev”
+mode, chaincode is built and started by the user. This mode is useful
+during the chaincode development phase for rapid code/build/run/debug cycle
+turnaround.
+
+- Note: Make sure the peer is not using TLS when running in dev mode.
+
+All commands are executed from the ``fabric`` folder.
+
+Setup environment
+-----------------
+Build binaries for orderer, peer and configtxgen.
+::
+    make orderer peer configtxgen
+::
+
+Setup PATH environment variable to include orderer and peer binaries.
+::
+    export PATH=$(pwd)/build/bin:$PATH
+::
+
+Setup FABRIC_CFG_PATH to point at sampleconfig.
+::
+    export FABRIC_CFG_PATH=$(pwd)/sampleconfig
+
+::
+
+Generate genesisblock.
+::
+    configtxgen -profile SampleDevModeSolo -channelID syschannel -outputBlock genesisblock -configPath $FABRIC_CFG_PATH -outputBlock $(pwd)/sampleconfig/genesisblock
+::
+
+Start the orderer
+-----------------
+
+::
+
+    ORDERER_GENERAL_GENESISPROFILE=SampleDevModeSolo orderer
+
+The above starts the orderer with the SampleDevModeSolo profile.
+
+Start the peer in dev mode
+--------------------------
+
+::
+
+    CORE_PEER_CHAINCODELISTENADDRESS=0.0.0.0:7052 peer node start --peer-chaincodedev=true
+
+The above command starts the peer using the default MSP.
+The ``--peer-chaincodedev=true`` puts it in “dev” mode.
+
+Create channel ch1
+---------------------------
+
+Generate the transactions for creating the channels using the ``configtxgen``
+tool with the SampleSingleMSPChannel profile.
+
+::
+
+   configtxgen -channelID ch1 -outputCreateChannelTx ch1.tx -profile SampleSingleMSPChannel -configPath $FABRIC_CFG_PATH
+
+::
+
+    peer channel create -o 127.0.0.1:7050 -c ch1 -f ch1.tx
+
+The above command assumes the orderer is reachable at ``127.0.0.1:7050``. The orderer
+is now tracking channel ch1 with the default configuration.
+
+::
+
+    peer channel join -b ch1.block
+
+The peer has now joined channel ch1.
+
+Build the chaincode
+-----------------------------
+
+The `simple` chaincode from the `integration/chaincode` directory is used here as an example.
+
+::
+    go build -o simpleChaincode ./integration/chaincode/simple/cmd
+::
+
+Start the chaincode
+-------------------
+
+::
+    CORE_CHAINCODE_LOGLEVEL=debug CORE_PEER_ADDRESS=192.168.0.7:7052 CORE_PEER_TLS_ENABLED=false CORE_CHAINCODE_ID_NAME=mycc:1.0 ./simpleChaincode -peer.address 192.168.0.7:7052
+
+The chaincode is started with the peer and chaincode logs indicating successful registration with the peer.
```

Comment:
```suggestion
This starts the chaincode and connects it to the peer. Look at the peer and chaincode logs to determine if the registration was successful. 
```
Not exactly sure on the best wording and if this conveys what you're going for, but just a suggestion. Also, maybe we should include an example of the log message?

Reply:
changes applied, just one minor thing, with provided peer logging and chaincode, there will be no log that displayed on both peer and chaincode when starting the chaincode.

Reply:
I was just going with what your previous text said, "peer AND chaincode logs." :) Definitely correct it so it only mentions the log you had in mind.